src/frontend/screens/Onboarding/walletSelection/screens/import/components/ImportAndCreateOptions/index.tsx

```diff
@@ -27,30 +33,44 @@ export default function ImportAndCreateOptions({
   }
 
   return (
-    <div className={styles.importOptionsContainer}>
-      {importableBrowserOptions}
-      <ImportOption
-        override="create"
-        title={t(
-          'hyperplay.onboarding.walletSelection.screens.import.createNewWallet',
-          `Create new MM extension wallet`
+    <>
+      <div className={styles.importOptionsContainer}>
+        {importableBrowserOptions}
+        {isOpenAdvancedOptions && (
```

Comment:
can we make this a ternary https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8